pkg/distributors/contracts/MultiDistributor.sol

```diff
@@ -704,8 +706,9 @@ contract MultiDistributor is IMultiDistributor, ReentrancyGuard, MultiDistributo
         UserDistribution storage userDistribution,
         uint256 updatedGlobalTokensPerStake
     ) internal view returns (uint256) {
+        // Note `unaccountedTokensPerStake` is a fixed point value
```

Comment:
```suggestion
        // Note `updatedGlobalTokensPerStake` is a fixed point value
```

We could also mention below that the fixed point multiplication below results in a non-fixed point value

Reply:
We could also mention somewhere (perhaps with the data structure description) that both 'tokens per stake' (user and global) are fixed point.